Formula/lib/libdpp.rb

```diff
@@ -0,0 +1,27 @@
+class Libdpp < Formula
+  desc "C++ Discord API Bot Library"
+  homepage "https://github.com/brainboxdotcc/DPP"
+  url "https://github.com/brainboxdotcc/DPP/releases/download/v10.0.28/DPP-10.0.28.tar.gz"
+  sha256 "aa0c16a1583f649f28ec7739c941e9f2bf9c891c0b87ef8278420618f8bacd46"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "libsodium"
+  depends_on "openssl@3"
+  depends_on "opus"
+  depends_on "pkg-config"
+
+  def install
+    system "cmake", "-B", "build", *std_cmake_args
+    cd "build" do
```

Comment:
See other formula with cmake to learn how to use the modern CMake build syntax

Reply:
Okie :)

Reply:
This is still pending

Reply:
Oh wait I'm confused. I looked at other formulas and I couldn't see what you meant by "modern CMake build syntax". I googled it and it came up saying this was the modern syntax and that something like `cmake ..` is old.

Reply:
This is the modern way that almost all formulae use.
```rb
system "cmake", "-S", ".", "-B", "build", *cmake_args
system "cmake", "--build", "build"
system "cmake", "--install", "build"
```

Reply:
Ohhhh, you meant the install part, I was so confused 😅 I'll fix that up for ya!

Reply:
I've submitted that now!